core/unit_test/CMakeLists.txt

```diff
@@ -759,6 +759,13 @@ if(Kokkos_ENABLE_CUDA)
     ${Cuda_SOURCES_SHAREDSPACE}
   )
 
+  # Suppress warning:
+  #   "Stack size for entry function 'xxx' cannot be statically determined"
+  # See https://github.com/kokkos/kokkos/issues/2039.
+  if(KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE)
+    target_link_options(Kokkos_CoreUnitTest_Cuda3 PRIVATE -Xnvlink -suppress-stack-size-warning)
+  endif()
```

Comment:
Was that warning actually turned into an error?
If so, I am leaning towards using CMAKE_EXE_LINKER_FLAG instead.

Not that this had been proposed and rejected in the past https://github.com/kokkos/kokkos/pull/4642#issuecomment-1006021381

Reply:
It generated a `nvlink` **error**. OK to pass it in the `.jenkins` through `-DCMAKE_EXE_LINKER_FLAG="-Xnvlink -suppress-stack-size-warning"` but beware it will be applied to *all* targets.

Reply:
Quoting from a failed job that does not have this `nvlink` option:
```bash
nvlink error   : Stack size for entry function '_ZN6Kokkos4Impl72_GLOBAL__N__48_tmpxft_00001f73_00000000_7_TestCuda_Task_cpp1_ii_6ea0a04923cuda_task_queue_executeINS_18BasicTaskSchedulerINS_4CudaENS0_9TaskQueueIS4_NS_12CudaUVMSpaceEEEEEEEvT_i' cannot be statically determined
```